packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/examples/EditableTable.js

```diff
@@ -0,0 +1,308 @@
+
+    this.state = {
+      columns: [
+        {
+          title: 'Repositories',
+          cellFormatters: [textInputFormatter]
+        },
+        {
+          title: 'Branches',
+          cellFormatters: [(value, { rowData }) => rowData.data.branches[rowData.data.branches.length - 1]]
+        },
+        'Pull requests',
+        {
+          title: 'Workspaces',
+          cellFormatters: [workspacesFormatter],
+          data: {
+            dropdownItems: ['Green', 'Purple', 'Orange', 'Grey']
+          }
+        },
+        {
+          title: 'Last Commit',
+          cellFormatters: [textInputFormatter]
+        },
+        {
+          title: 'Private',
+          cellFormatters: [privateRepoFormatter]
+        }
+      ],
+      rows: [
+        {
+          cells: ['one', null, 7, null, 'five', null],
+          data: {
+            branches: ['master'],
+            workspace: {
+              selected: 'Green',
+              isDropdownOpen: false
+            },
+            privateRepo: false
+          }
+          // isEditing: true,
+        },
+        {
+          cells: ['', null, 0, null, '', null],
+          data: {
+            branches: ['master', 'v0.7.0', 'v1.0.0'],
+            workspace: {
+              selected: 'Grey',
+              isDropdownOpen: false
+            },
+            privateRepo: false
+          }
+        },
+        {
+          cells: ['p', null, 0, null, '', null],
+          data: {
+            branches: ['master', 'v0.7.0'],
+            workspace: {
+              selected: 'Orange',
+              isDropdownOpen: false
+            },
+            privateRepo: true
+          }
+        }
+      ],
+      actions: [
+        {
+          title: 'Edit',
+          onClick: this.onEditActionClick
+        }
+      ],
+      editedRowBackup: null,
+      activeEditId: null
+    };
+  }
+
+  onPrivateRepoChange = (value, { rowIndex }) => {
+    this.setState(({ rows }) => {
+      const row = rows[rowIndex];
+      row.data.privateRepo = value;
+      return { rows };
+    });
+  };
+
+  onWorkspaceChange = (value, { rowIndex, columnIndex }) => {
+    this.setState(({ rows, activeEditId }) => {
+      const row = rows[rowIndex];
+      row.data.workspace = Object.assign({}, row.data.workspace, value);
+      if (value.isDropdownOpen) {
+        activeEditId = this.makeId({ rowIndex, columnIndex, column: { property: 'workspaces' }, name: 'dropdown' });
+      }
+      return { rows, activeEditId };
+    });
+  };
+
+  onChange = (value, { rowIndex, columnIndex }) => {
+    this.setState(({ rows }) => {
+      rows = [...rows];
+      const row = rows[rowIndex];
+      row.cells[columnIndex] = value;
+      return {
+        rows,
+        activeEditId: null // stop autoFocus
+      };
+    });
+  };
+
+  onEditCellClicked = (event, clickedRow, { rowIndex, columnIndex, elementId }) => {
+    const WORKSPACE_COL = 3;
+    const PRIVATE_REPO_COL = 5;
+    const ACTIONS_COL = 6;
+
```

Comment:
the consumer must supply the logic what to do with the click and can work with the ids